src/test/java/LottoTest.java

```diff
@@ -0,0 +1,28 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+class LottoTest {
+    @Test
+    void Lotto를_추가할_수_있다() {
+        Lotto lotto = new Lotto();
+        assertDoesNotThrow(() -> {
+            lotto.add(new LottoNumbers(
+                    new LottoNumber(1),
+                    new LottoNumber(2),
+                    new LottoNumber(3),
+                    new LottoNumber(4),
+                    new LottoNumber(5),
+                    new LottoNumber(6)
+            ));
+        });
+    }
+
+    @Test
+    void Lotto_의_크기를_알_수_있다() {
+        Lotto lotto = new Lotto();
+        assertThat(lotto.size()).isEqualTo(0);
```

Comment:
지금 코드도 나쁘진 않지만 좀 더 명확하게 한다면 이렇게도 할 수 있을것 같아요!
```suggestion
        assertThat(lotto.size()).isZero();
```

Reply:
@wooobo 
네. 🙆  매직 넘버는 최소화 하는게 저도 좋다고 생각합니다.